### PR TITLE
GEOT-5793 MultiGeometry not working in GML 3.2 parser/encoded

### PR DESCRIPTION
**Pull request by NielsCharlier**

### CODE REVIEW

**Comment by bencaradocdavies** [comment]
Jira issue?

File: modules/extension/xsd/xsd-gml3/src/main/java/org/geotools/gml3/bindings/MultiGeometryTypeBinding.java [hunk 1]
```diff
@@ -112,7 +112,7 @@ public Object parse(ElementInstance instance, Node node, Object value)
     
     @Override
     public Object getProperty(Object object, QName name) throws Exception {
-        if (GML.geometryMember.equals(name)) {
+        if ("geometryMember".equals(name.getLocalPart())) {
```
Comment (bencaradocdavies) [inline modules/extension/xsd/xsd-gml3/src/main/java/org/geotools/gml3/bindings/MultiGeometryTypeBinding.java]: Bare string "geometryMember"? How about `GML.geometryMember.getLocalPart().equals(name.getLocalPart())`? Is this change to make it work for both GML 3.1 and 3.2?

**Comment by bencaradocdavies** [comment]
@NielsCharlier this looks good and seems to make sense but needs a Jira issue.

**Comment by NielsCharlier** [comment]
Turns out I did make a jira issue, I just forgot to mention it here GEOT-5793

File: modules/extension/xsd/xsd-gml3/src/main/java/org/geotools/gml3/bindings/MultiGeometryTypeBinding.java (hunk 1 above)
Comment (NielsCharlier) [inline modules/extension/xsd/xsd-gml3/src/main/java/org/geotools/gml3/bindings/MultiGeometryTypeBinding.java]: Yes, I followed the same pattern of all the other classes which use literals.
But I agree that would be better.

**Comment by NielsCharlier** [comment]
Thanks for your review @bencaradocdavies 
I have made the suggested change

**Comment by bencaradocdavies** [comment]
Looks good @NielsCharlier.

**Pull request merged**